### PR TITLE
Standardize examples to use @fluidframework/azure-local-service instead of tinylicious

### PR DESCRIPTION
**Pull request by scottn12**
Standardizes all examples to use `@fluidframework/azure-local-service` instead of tinylicious

### CODE REVIEW

**Comment by ChumpChief** [comment]
We should probably normalize on `@fluidframework/azure-local-service` since that's what we advertise to customers to use.

**Comment by scottn12** [comment]
> We should probably normalize on `@fluidframework/azure-local-service` since that's what we advertise to customers to use.

@ChumpChief, do you know where we advertise to customers to use azure-local-service? I can't really find it on ff.com and it looks like we encourage using either tinylicious or just FRS [here](https://fluidframework.com/docs/deployment/service-options/).

**Comment by ChumpChief** [comment]
> @ChumpChief, do you know where we advertise to customers to use azure-local-service? I can't really find it on ff.com and it looks like we encourage using either tinylicious or just FRS [here](https://fluidframework.com/docs/deployment/service-options/).

https://learn.microsoft.com/en-us/azure/azure-fluid-relay/how-tos/local-mode-with-azure-client#running-azure-fluid-relay-service-locally

File: angular-demo/package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
 		"@angular-devkit/build-angular": "~13.3.9",
 		"@angular/cli": "~13.3.9",
 		"@angular/compiler-cli": "~13.3.11",
+		"@fluidframework/azure-local-service": "^1.1.0",
```
Comment (ChumpChief) [inline angular-demo/package.json:43]: If we're doing npx, then shouldn't need to include this as a dependency, same for other instances in other demos.

**Review by ChumpChief — APPROVED** [review]

**Pull request merged**